docs_src/OVMS/pipelinerun.md

```diff
@@ -0,0 +1,141 @@
+# Run Pipeline
+
+## Prerequisites 
+Before running, [set up the pipeline](./pipelinesetup.md).
+
+---
+## Overview 
+When the pipeline is run, the `docker-run.sh` script starts the service and performs inferencing on the selected input media. The output of running the pipeline provides the inference results for each frame based on the media source such as text, barcode, and so on, as well as the frames per second (FPS). Pipeline run provides many options in media type, system process platform type, and additional optional parameters. These options give you the opportunity to compare what system process platform is better for your need.
+
+## Start Pipeline
+
+You can run the pipeline script, `docker-run.sh`, with the following input parameters:
```

Comment:
i think it is important to mention when ovms docker run to use --workload parameter explicitly for ovms as the default for that is dlstreamer.